editor/src/core/shared/template-path.ts

```diff
@@ -295,8 +295,19 @@ export function instancePathForElementAtScenePath(path: ScenePath): StaticInstan
   }
 }
 
+function instancePathForElementPaths(elementPaths: ElementPath[]): InstancePath {
+  const lastElementPath = last(elementPaths)
```

Comment:
this function should share logic with `fromStringUncached`!

fromStringUncached should be the same as `instancePathForElementPaths(string.split(':').map(elementPath => elementPath.split('/'))`

toString should just be `instancePathToElementPaths().join('/').join(':')`

Reply:
Yep, agreed. But I think that would be better placed to happen alongside the removal of `ScenePath` and `InstancePath` otherwise it risks introducing regressions elsewhere.

Reply:
I didn't run the code, but reading it I think the logic doesn't match fromString now, if the elementPaths is a single array long, that should become a ScenePath and no instance part

Reply:
in that case, I would make it return emptyinstance path if the elementPaths is 0 or 1 long!

Reply:
No, because that's not what this is for. This function is for returning the InstancePath for a given ElementPath array, to help bridge the gap until we remove InstancePath and ScenePath.